CONTRIBUTING.md

```diff
@@ -3,125 +3,96 @@
 
 # Contributing
 
-Please note we have a [code of conduct](https://github.com/hotwired/turbo/blob/main/CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
+Note that we have a [code of conduct](https://github.com/hotwired/turbo/blob/main/CODE_OF_CONDUCT.md). Please follow it in your interactions with this project.
 
 ## Sending a Pull Request
 
 The core team is monitoring for pull requests. We will review your pull request and either merge it, request changes to it, or close it with an explanation.
 
-Before submitting a pull request, please make sure the following is done:
+Before submitting a pull request, please:
 
-1. Fork the repository and create your branch from main.
-2. Run `yarn` in the repository root.
-3. If you’ve fixed a bug or added code that should be tested, add tests!
-4. Ensure the test suite passes (`yarn build && yarn test`).
+1. Fork the repository and create your branch.
+2. Follow the setup instructions in this file.
+3. If you’re fixing a bug or adding code that should be tested, add tests!
+4. Ensure the test suite passes.
 
 ## Developing locally
 
-During the process of developing the library locally we first have to check out the repository and create a branch from main.
+First, clone the `hotwired/turbo` repository and install dependencies:
 
 ```bash
 git clone https://github.com/hotwired/turbo.git
-cd turbo
-yarn
 ```
 
 ```bash
-git checkout -b '<your_branch_name>'
+cd turbo
+yarn install
 ```
 
-Once you are done developing the feature or bug fix you have 2 options:
+Then create a branch for your changes:
 
-1. Run the test suite
-2. Run a local webserver and checkout your changes manually
+```bash
+git checkout -b <your_branch_name>
+```
 
 ### Testing
 
-The library is tested by running the test suite (found in: `src/tests/*`) against headless browsers. The browsers are setup in [web-test-runner.config.mjs](./web-test-runner.config.mjs) and [playwright.config.ts](./playwright.config.ts). Check them out to see the used browser environments.
+Tests are run through `yarn` using [Web Test Runner](https://modern-web.dev/docs/test-runner/overview/) with [Playwright](https://github.com/microsoft/playwright) for browser testing. Browser and runtime configuration can be found in [`web-test-runner.config.mjs`](./web-test-runner.config.mjs) and [`playwright.config.ts`](./playwright.config.ts).
 
-To override the ChromeDriver version, declare the `CHROMEVER` environment
-variable.
```

Comment:
Removed this reference to `CHROMEVER` as it appears to be unused since https://github.com/hotwired/turbo/pull/866 moved from Intern to WTR.